app/src/main/res/layout-land/fragment_iab.xml

```diff
@@ -7,7 +7,11 @@
     android:background="@drawable/background_card"
     >
 
-  <include layout="@layout/fragment_iab_loading"/>
-  <include layout="@layout/fragment_iab_transaction_completed"/>
-  <include layout="@layout/fragment_iab_error"/>
+  <include layout="@layout/fragment_iab_loading" />
+  <include layout="@layout/fragment_iab_transaction_completed" />
+  <include
+      android:id="@+id/something"
```

Comment:
please rename this id to something meaningful

Reply:
debug stuff XD